src/platform/testing/e2e/cypress/support/form-tester/index.js

```diff
@@ -228,7 +228,7 @@ const defaultPostHook = pathname => {
   return () => {
     cy.findByText(/continue/i, {
       selector: 'button',
-      timeout: 20000,
+      timeout: 30000,
     })
```

Comment:
Without increasing this timeout again Cypress tests unrelated to this application were failing on Github.

Reply:
Haven't we already increased it once? I'm starting to see why our tests fail randomly.

Reply:
Yep, and I've informed the individuals that helped discover this was an issue the first time that I've continued to have to up this number.